Library/Formula/fontforge.rb

```diff
@@ -1,7 +1,9 @@
 class Fontforge < Formula
   homepage "https://fontforge.github.io"
+  # We use the archive tarball because it's smaller and compiles fine.
```

Comment:
@adamv The 255MB "release" tarball. Talked to upstream about this pretty extensively in the past and more recently, and their opinion essentially is if we can use the smaller tarball and end up with the same end-result, which we do, saving users from a >250MB download is only a good thing :smile_cat:.

@mikemcquaid Will chop it.